console.rb

```diff
@@ -3,15 +3,43 @@
 require_relative "models/tenant"
 
 # NOTE: Uncomment the lines below to verify that your seed script is working
-
+#
 # puts "There are #{Apartment.count} apartments"
 # puts "There are #{Tenant.count} tenants"
-#
 # puts "*" * 50
-#
 # puts "The first apartment is at #{Apartment.first.address}."
 # puts "It has  #{Apartment.first.tenants.count} tenants."
 
+value = 0
+while value < 5 do
+puts "Welcome, landlord! To see a list of all your apartments, press 1. \n
+To see a list of all your tenants, press 2.\n
+To see a list of apartments and their tenants, press 3"
+response = gets.chomp.to_i
```

Comment:
You started your `while` loop expecting the input variable to be `value`, but then switched to `response` here. This technically works, but typing a number assigns  `response` only and `value` never gets changed. This means the the loop runs forever because `value` is always 0. The simplest fix is to change `value` to `response`:
```ruby
response = 0
while response < 5 do
```